schemas/lakehouse/aep-lakehouse-batch-errors.schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/schemas/aep-lakehouse-batch-errors",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AEP Lakehouse Batch Errors",
+  "description": "Global schema for AEP Lakehouse datasets containing batch errors for diagnostics.",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "definitions": {
+    "batchInputFiles": {
+      "properties": {
+        "xdm:datasetId": {
+          "title": "DatasetId",
+          "description": "Dataset Id the batch was created for.",
+          "type": "string"
+        },
+        "xdm:batchId": {
+          "title": "BatchId",
+          "description": "Id for the batch containing the input files.",
+          "type": "string"
+        },
+        "xdm:path": {
+            "title": "Path",
+            "description": "Path to the input file that contained this record.",
+            "type": "string"
+        },
+        "xdm:record": {
+          "title": "Record",
+          "description": "JSON encoded record that failed to process.",
+          "type": "string"
+        },
+        "xdm:corruptRecord": {
+          "title": "CorruptRecord",
+          "description": "Original record that failed to process",
+          "type": "string"
+        },
+        "xdm:type": {
```

Comment:
I'll recommend to change the field name to `errorType` to avoid collision with the reserved keyword `type` of JSON Schema. When reading the schema from XDM registry, the prefix "xdm:" is dropped and the field will be returned as `type`, which could have unintended consequences downstreams.